bin/node/src/service.rs

```diff
@@ -25,12 +25,33 @@ use sp_runtime::{
     traits::{Block as BlockT, Header as HeaderT, Zero},
 };
 
-use crate::{aleph_cli::AlephCli, executor::AlephExecutor};
+use crate::{
+    aleph_cli::{AlephCli, DEFAULT_BACKUP_FOLDER},
+    executor::AlephExecutor,
+};
 
 type FullClient = sc_service::TFullClient<Block, RuntimeApi, AlephExecutor>;
 type FullBackend = sc_service::TFullBackend<Block>;
 type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
 
+fn get_backup_path(aleph_config: &AlephCli, config: &Configuration) -> Option<PathBuf> {
+    if aleph_config.no_backup() {
+        return None;
+    }
+    if let Some(path) = aleph_config.backup_path() {
+        Some(path)
+    } else {
+        let path = config
+            .base_path
+            .as_ref()
+            .expect("Please specify base path")
+            .path()
```

Comment:
this unwrapping seems a bit unrelated to the `get_backup_path` semantics - I would pass to this method ready `base_path` instead of whole `Configuration` object

Reply:
done